locales/en/openbridge.json

```diff
@@ -13,12 +13,14 @@
     "details": "Details",
     "edit": "Edit",
     "endpoint": "Endpoint",
+    "genericError": "Generic Error",
     "id": "ID",
     "kafkaTopic": "Kafka Topic",
     "key": "Key",
     "loading": "Loading",
     "name": "Name",
     "owner": "Owner",
+    "pleaseCorrectError": "Please correct the following errors before continuing.",
```

Comment:
According to mockups, this should be `Address form errors to proceed.`

Reply:
Ok, I'll change. Probably lost the mockups you're mentioning. Can you please link them? Thank you

Reply:
Sure, here's the [create instance deck](https://docs.google.com/presentation/d/1TCuQdKiS282CyK7m6Mnepm1jzX9cCT2rg2R5_NjgXeo/edit#slide=id.g124f01a706d_0_8).